lib/registrant.rb

```diff
@@ -0,0 +1,20 @@
+class Registrant
+    attr_reader :name, :age, :permit
+    attr_accessor :license_data
```

Comment:
For the purposes of this project, using an `attr_accessor` is just fine. Generally, though, we want to stay away from using these because they allow outside classes to mutate our data. Another alternative to using `attr_accessor` here is to create `setter_methods`. For example, if a registrant takes the written exam, you could create a method here called `pass_written_exam` and inside that method, you'd set `licence_data[:written] = true`. In this way, we can control the setting of this value to make sure it's not something random (like it won't end up being `license_data[:written] = yes` or something inconsistent). We could also add controls to this method, for example, if we got a score argument, we could only set the `true` value `if score > 80`. Using a setter method gives us more control over setting this value rather than letting outside code change it to whatever it wants. 

Again, this simple of a case is no problem. Just wanted to share this context because you may hear discussion of why attr_accessors are not always the best choice